src/core/server/elasticsearch/cluster_client.ts

```diff
@@ -159,36 +158,24 @@ export class ClusterClient {
   };
 
   /**
-   * Closes the cluster client. After that client cannot be used and one should
-   * create a new client instance to be able to interact with Elasticsearch API.
-   */
-  public close() {
-    if (this.isClosed) {
-      return;
-    }
-
-    this.isClosed = true;
-    this.client.close();
-
-    if (this.scopedClient !== undefined) {
-      this.scopedClient.close();
-    }
-  }
-
-  /**
-   * Creates an instance of `ScopedClusterClient` based on the configuration the
-   * current cluster client that exposes additional `callAsCurrentUser` method
-   * scoped to the provided req. Consumers shouldn't worry about closing
-   * scoped client instances, these will be automatically closed as soon as the
-   * original cluster client isn't needed anymore and closed.
-   * @param request - Request the `ScopedClusterClient` instance will be scoped to.
-   * Supports request optionality, Legacy.Request & FakeRequest for BWC with LegacyPlatform
+   * Calls specified `endpoint` with provided `clientParams` on behalf of the
+   * user initiated request to the Kibana server (via HTTP request headers).
+   * @param request - Request to borrow HTTP headers from
+   * @param endpoint - String descriptor of the endpoint e.g. `cluster.getSettings` or `ping`.
+   * @param clientParams - A dictionary of parameters that will be passed directly to the Elasticsearch JS client.
+   * @param options - Options that affect the way we call the API and process the result.
    */
-  public asScoped(request?: KibanaRequest | LegacyRequest | FakeRequest) {
```

Comment:
Should we tag it as a breaking change? I found only `Security` plugin uses this API, although I could miss a plugin that is in progress of the migration.

Reply:
Yes we definitely should. There are at least 3 or 4 new in-development plugins. Let's add it to this issue #40768

Reply:
done